### ISSUE TITLE
How to get started?

### ISSUE THREAD

**Issue by mariusa** [Redocly/redoc#1269]
Hi,

I'm trying to get to a result similar to https://open-api.netlify.com/
(a set of static files that can be deployed anywhere)

1. The homepage link at the top of README https://github.com/Redocly/redoc
is just a sample doc output? (pet store) Really confusing for new comers, should be in Some Real-life usages section. 

2. Deployment TL;DR 
and ```redoc-cli bundle pet.json``` work

but how to generate spec.json file from a set of markdown files?

Installed and ran ```create-openapi-repo```
This creates a .yaml file without asking for json or yaml preference.

But then, should we edit the .yaml file manually? 

Thanks

**Comment by c3rajkaramchedu** [comment]
@mariusa Yes, you would need to edit the YAML file manually, using a text editor. I personally prefer YAML format with ReDoc. The easier way I found to learn this is to download the spec, which is in YAML format, from here: http://redocly.github.io/redoc/ and start comparing the YAML text file structure with the rendered HTML doc. Alternately, a few weeks ago I found this but haven't really used it. Seems like a useful tool: https://stoplight.io/studio/.
